data/json/npcs/robofac/NPC_robofac_generic_security.json

```diff
@@ -52,12 +57,37 @@
     "subtype": "collection",
     "entries": [
       { "item": "grenade_emp", "prob": 33 },
-      { "item": "grenade", "prob": 2 },
-      { "item": "PR24-extended" },
-      { "item": "flashlight" },
-      { "item": "glock_17", "charges": 21, "ammo-item": "9mmP" },
-      { "item": "glock17_22", "charges": 21, "ammo-item": "9mmP" },
-      { "item": "9mmP", "count": 6 }
+      {
+        "collection": [
+          { "item": "glock_17", "charges": 22, "ammo-item": "9mmP" },
+          { "item": "glock17_22", "charges": 22, "ammo-item": "9mmP", "count": [ 1, 2 ] },
+          { "item": "9mmP", "count": [ 1, 2 ] },
+          { "item": "stanag30", "charges": 30, "ammo-item": "556", "count": [ 1, 2 ] },
+          { "item": "556", "count": 2 }
+        ]
+      },
+      { "item": "grenade", "prob": 2 }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "NC_robofac_security_weapon",
+    "subtype": "collection",
+    "entries": [ { "item": "m27_assault_rifle", "variant": "m38dmr", "contents-group": "robofac_sec_weapon_mods", "charges": 30 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "robofac_sec_weapon_mods",
+    "subtype": "collection",
+    "entries": [
+      {
+        "distribution": [
+          { "item": "holo_sight", "prob": 50 },
+          { "item": "acog_scope", "prob": 30 },
+          { "item": "hybrid_sight_4x", "prob": 30 }
```

Comment:
to be picky the hybrid 4x should probably also include a red dot / holo as a spawn group since thats its main upside, that it can accept a second sight.

Reply:
I didn't want to go too overboard with the mods, that said I would like to make them look cool, giving them cool tactical equipment is a thing I want to do
dressing up npcs is very fun

ahh never mind, this is the special acog scope, I don't think this should go here...

Reply:
Well it's your choice but for the record sights like that exist IRL it isn't sci-fi or speculative so if you are ever dressing up NPCs no need to be too conservative with optics. 

https://www.trijicon.com/products/product-family/trijicon-acog-rmr-combo

 https://huntingmark.com/leupold-hamr-review/

Are examples